### DISCUSSION TITLE
where is my ipa?

### DISCUSSION THREAD

**Discussion by WTFCSL** [fastlane/fastlane#16959] [Q&A]
>

>
![image](https://user-images.githubusercontent.com/10007401/88681987-b8f03800-d124-11ea-8c3c-4da9c58baae0.png)

action：upload_to_app_store
options：ipa （ipa path）
it's invalid

**Comment by BarbaraBryant** [comment]
You say not appdb but it’s the only one with a data base that big and it’s the only one i trust (if you download from verified crackers).[FMC 4 ME](https://www.fmc4me.net/)
